jib-core/src/main/java/com/google/cloud/tools/jib/json/JsonTemplateMapper.java

```diff
@@ -109,7 +109,7 @@
    * @return the template filled with the values parsed from {@code jsonFile}
```

Comment:
comment `{@code jsonFile} -> {@code jsonString}`? Also a little curious what this is doing? Does this just parse a string that we know is a list?

Reply:
Yep, this would try to parse a string as a JSON list. It would fail if it is not a list.

Reply:
Is there a reason we can just use `readJson`?

Reply:
Filed #997

Reply:
If feasible, can we fix #997 first and add "not ready for review" to this PR? Seems like #997 has a significant cleanup impact on this. Or, will it be too much work?

Reply:
Not sure if #997 is valid to do yet. We'll have to explore that more and see if things will work without `ListOfJsonTemplate`, but I don't think it'll affect this PR much since the serialization of `LayerEntryTemplate` can just store it in a java List.